env/servers.env.example

```diff
@@ -0,0 +1,8 @@
+DEV_EMAIL_ADDRESS=
```

Comment:
I reckon we might wanna move this under `mail.env`. We only use it when we're sending an email to ensure non-production environments don't send emails to external users